dask_expr/_util.py

```diff
@@ -18,6 +19,10 @@ def _convert_to_list(column) -> list | None:
     return column
 
 
+def is_scalar(x):
+    return not (isinstance(x, Sequence) or hasattr(x, "dtype")) or isinstance(x, str)
```

Comment:
Any reason not to use `np.isscalar`?

Reply:
Okay, fair enough. If you can think of the case, it might be good to document in a comment. Otherwise, someone (probably me) will end up using `np.isscalar` in the wrong place, or will ask this same question again :)

Reply:
```
import pandas as pd
import numpy as np
np.isscalar(pd.NA)
```

This returns False. I'll add a comment